datalad/distribution/publish.py

```diff
@@ -60,23 +60,23 @@ class Publish(Interface):
         # Note: add remote currently disabled in publish
         # dest_url=Parameter(
         #     args=('--dest-url',),
-        #     doc="""The URL of the dataset sibling named by `dest`. This URL has
+        #     doc="""the URL of the dataset sibling named by `dest`.  This URL has
         #     to be accessible to anyone, who is supposed to have access to the
         #     published dataset later on.\n
         #     If you want to publish with `recursive`, it is expected, that you
```

Comment:
@yarikoptic this is after newline - I left it capitalized since it's still withing in the same doc string. Is that ok? Couldn't find an example on cmdline to replicate the situation.